crates/eigen-cli/src/eigen_address.rs

```diff
@@ -0,0 +1,212 @@
+use crate::args::Args;
+use crate::EigenAddressCliError;
+use alloy_contract::Error as ContractError;
+use alloy_primitives::Address;
+use alloy_provider::Provider;
+use eigen_utils::{
+    binding::{DelegationManager, IBLSSignatureChecker, RegistryCoordinator},
+    get_provider,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct EigenAddresses {
```

Comment:
it's confusing to have an EigenAddresses and an EigenLayerAddresses stuct, could we rename that?